TeamViewerADConnector/Internal/Sync.ps1

```diff
@@ -195,9 +195,13 @@ function Invoke-SyncUser($syncContext, $configuration, $progressHandler) {
                 if ($configuration.UseSsoCustomerId) {
                     $newUser.sso_customer_id = $configuration.SsoCustomerId;
                 }
+                if ($configuration.MeetingLicenseKey) {
+                    $newUser.meeting_license_key = $configuration.MeetingLicenseKey;
+                }
                 try {
                     $addedUser = (Add-TeamViewerUser $configuration.ApiToken $newUser)
                     $newUser.id = $addedUser.id
+                    $newUser.MeetingLicenseKey = $addedUser.activated_meeting_license_key
```

Comment:
why is this necessary? The data is not shown anywhere in the logs, isn't it?
If really needed, I'd suggest to rename this to `ActivatedMeetingLicenseKey`

Reply:
I somehow ended up with the impression, that the user data is returned to the caller.
As it's actually not used at the moment I have removed this line (and adapted the test) .. but I will think about potentially adding a log entry for cases where the assignment failed.